backend/src/test/java/com/google/slurp/ConvertEmailsToUidsTest.java

```diff
@@ -0,0 +1,100 @@
+package com.google.slurp;
+
+import java.io.BufferedReader;
+import java.io.PrintWriter;
+import java.io.StringReader;
+import java.io.StringWriter;
+
+import javax.servlet.ServletContextEvent;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import com.google.slurp.servlets.ConvertEmailsToUidsServlet;
+import com.google.slurp.servlets.ServerListener;
+
+import org.junit.AfterClass;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import org.mockito.Mockito;
+
+/** */
+@RunWith(JUnit4.class)
+public class ConvertEmailsToUidsTest extends Mockito {
+  // These are the test accounts registered with Firebase Authentication.
+  private static final String USER_1_EMAIL = "\"step2020.53.test1@gmail.com\"";
+  private static final String USER_2_EMAIL = "\"step2020.53.test2@gmail.com\"";
+  private static final String USER_1_UID = "\"Eq00IcXeNbfwgL8ZAIHw4Alp9i53\"";
+  private static final String USER_2_UID = "\"IjtXS7Py9ZeorDRPmsY52RBWhop1\"";
+
+  // Lists of UIDs and expected lists of emails, constructed from the test users.
```

Comment:
I thought that the 2-email case would've been sufficient since 1-email is still just an array that's being acted upon, just with 1 email. I went ahead and added test cases for the 1-email case though.